docs/guides/workflows_patterns/deployment_astronomer.rst

```diff
@@ -0,0 +1,55 @@
+.. _deployment_astronomer:
+
+Deploying Great Expectations with Astronomer
+=============================================
+
+This guide will help you deploy Great Expectations within an Airflow pipeline that runs in `Astronomer<https://www.astronomer.io/>`_.  You can use Great Expectations to automate validation of data integrity and navigate your DAG based on the output of validations.
+
+.. admonition:: Prerequisites: This workflow pattern assumes you have already:
+
+    - Followed the instructions for :ref:`creating a Great Expectations validation task in an Airflow DAG<deployment_airflow>`
+    - Set up an `Astronomer deploy<https://www.astronomer.io/docs/cloud/stable/develop/cli-quickstart/>`_
+
+Using the Great Expectations Airflow Operator in an Astronomer Deployment
+=========================================================================
```

Comment:
Using the `=` underline gives this section it's own separate link in the sidebar under "workflows & patterns" - is that intended?
```suggestion
-------------------------------------------------------------------------
=========================================================================
```

Reply:
Yeah, see the table of contents entry at the top - we figured since there's a Composer page, it makes sense to have a separate Astro page too.